articles/modules/ROOT/pages/setting-up-ssl-with-docker.adoc

```diff
@@ -10,10 +10,141 @@ Neo4j 3.2 added a Unified SSL Framework to setup secure connections for Bolt, HT
 framework can be found at:  https://neo4j.com/docs/operations-manual/current/security/ssl-framework/
 
 Setting up secure Bolt and HTTPS communications when running Neo4j within a Docker Container requires some specific steps and 
-configuration settings:
+configuration settings.
 
 The below steps assume you've followed the process outlined in the above documentation link and have created your public cert and
-private key.
+private key. 
+
+CAUTION: The steps listed below are for example purposes only. You should make sure these are suitable for your requirements before deploying into any environment. 
+
+The configuration steps for Neo4j 3.5 and Neo4j 4.x are different. Both are listed below.
+
+== Neo4j 4.x
+
+=== 1. Create Neo4j Scope Directory Structure
+
+Neo4j 4.x requires directories are created with a specific structure. The following in an example for `https` and `bolt` scopes
+
+----
+ssl/
+├── bolt
+│   ├── revoked
+│   └── trusted
+└── https
+    ├── revoked
+    └── trusted
+----
+
+Create the above folder structure in `$HOME/neo4j`. It is possible to use any directory of your choice, making sure to adjust the below paths accordingly.
+
+=== 2. Copy SSL Certificates and Correct Permissions
+
+A copy of your public certificate and private key will need to be placed in each scopes directory. The public key will also need to be copied into the respective scopes `trusted` folder.
+
+Once copied, your folder and file layout in `$HOME/neo4j/ssl` should look like the following:
+
+----
+$HOME/neo4j/ssl/
+├── bolt
+│   ├── private.key
+│   ├── public.crt
+│   ├── revoked
+│   └── trusted
+│       └── public.crt
+└── https
+    ├── private.key
+    ├── public.crt
+    ├── revoked
+    └── trusted
+        └── public.crt
+----
+
+The container will run Neo4j as user `neo4j`. The uid and gid is 7474. Use the following command to adjust the permissions on the `ssl` folder:
+
+[source,shell]
+----
+sudo chgrp -R 7474 $HOME/neo4j/ssl && \
+sudo chmod -R g+rx $HOME/neo4j/ssl
+----
+
+=== 3. Configure Neo4j
+
+There are two methods for configuring Neo4j running as a Docker container: *externalize neo4j.conf using Docker host volumes*, OR *configure Docker environment variables*. You should only choose one of the methods (not both)
+
+==== Externalize neo4j.conf
+
+It is possible to use a Docker host volume to mount a neo4j.conf file. This is achieved by mounting into the containers `/conf` directory.
+
+Create a `conf` directory and neo4.conf file in `$HOME/neo4j`
+
+[source,shell]
+----
+mkdir $HOME/neo4j/conf && \
+touch $HOME/neo4j/conf/neo4j.conf
+----
+
+Insert the following configuration into `neo4j.conf` making sure to substitute `host.domain.com` with your DNS name.
```

Comment:
Consider generalising statements by replacing "you" or "your" with e.g. "the desired" in this case